ajax/libs/shards-vue/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "shards-vue",
+  "filename": "dist/shards-vue.common.min.js",
+  "version": "1.0.3",
```

Comment:
```suggestion
```

Please remove the version attr, the auto-update bot will add this once it brings in versioned library files.